src/main/java/com/hugman/culinaire/rei/CulinairePlugin.java

```diff
@@ -0,0 +1,72 @@
+package com.hugman.culinaire.rei;
+
+import com.hugman.culinaire.init.CulinaireTeaBundle;
+import com.hugman.culinaire.objects.item.tea.TeaHelper;
+import com.hugman.culinaire.objects.item.tea.TeaType;
+import com.hugman.culinaire.objects.recipe.TeaBagMakingRecipe;
+import com.hugman.culinaire.objects.screen.KettleScreen;
+import me.shedaniel.math.Rectangle;
+import me.shedaniel.rei.api.client.REIRuntime;
+import me.shedaniel.rei.api.client.plugins.REIClientPlugin;
+import me.shedaniel.rei.api.client.registry.category.CategoryRegistry;
+import me.shedaniel.rei.api.client.registry.display.DisplayRegistry;
+import me.shedaniel.rei.api.client.registry.screen.ScreenRegistry;
+import me.shedaniel.rei.api.common.category.CategoryIdentifier;
+import me.shedaniel.rei.api.common.util.EntryStacks;
+import net.minecraft.item.ItemStack;
+import net.minecraft.recipe.*;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.collection.DefaultedList;
+
+public class CulinairePlugin implements REIClientPlugin {
```

Comment:
This class could be renamed `CulinaireREIPLugin`, so that we can introduce more plugins in the future and not be confused by the names